lib/movable_ink/aws/ssm.rb

```diff
@@ -3,12 +3,12 @@
 module MovableInk
   class AWS
     module SSM
-      def ssm_client
-        @ssm_client ||= Aws::SSM::Client.new(region: 'us-east-1')
+      def ssm_client(region = nil)
+        @ssm_client ||= (region) ? region : Aws::SSM::Client.new(region: 'us-east-1')
       end
 
-      def ssm_client_failover
-        @ssm_client_failover ||= Aws::SSM::Client.new(region: 'us-west-2')
+      def ssm_client_failover(failregion = nil)
+        @ssm_client_failover ||= (failregion) ? failregion: Aws::SSM::Client.new(region: 'us-west-2')
```

Comment:
Same here, `@ssm_client_failover` will get `us-east-1` if `ssm_client_failover('us-east-1')` is called.

Reply:
fixed using default arguments